tapir/coop/tests/membership_resignation/test_form.py

```diff
@@ -21,30 +26,102 @@ def setUp(self) -> None:
         self.given_feature_flag_value(feature_flag_membership_resignation, True)
         mock_timezone_now(self, self.NOW)
 
-    def test_empty_form(self):
-        form = MembershipResignationForm()
-        self.assertIn("share_owner", form.fields)
-        self.assertIn("resignation_type", form.fields)
-        self.assertIn("cancellation_date", form.fields)
-        self.assertIn("transferring_shares_to", form.fields)
+    def test_membershipResignationForm_is_valid(self):
```

Comment:
Try to stick with the naming convention described [here](https://github.com/SuperCoopBerlin/tapir/wiki/Automated-tests-%E2%80%90-overview#naming-convention).
Here for example, we already know that we are testing the form because of the test file name and the test class name. The thing you are testing is the "is_valid" function of the form. So the function could be called something like `test_isValid_sendingValidData_returnsTrue`.

I think the test would also be clearer with explicit values rather than values from the factory, so with `"cancellation_reason": "a test reason"`.